versions.properties

```diff
@@ -27,6 +27,10 @@ version.android.buildTools=34.0.0
 
 # Project dependencies
 
+version.kotlinx.coroutines=1.8.1
+##             # available=1.9.0-RC
+##             # available=1.9.0-RC.2
+
```

Comment:
doesnt seem like these changes are related?

Reply:
I don't think they are but I'm a bit unfamiliar with this tool - do we just manually remove these update lines?

Reply:
Hmm not sure where this woulda come from. Could this have bled over from your other branch about concurrent modification? Because this PR doesn't seem to have anything to do with coroutines

Reply:
these are ok to have but shouldn't be bundled with these changes to 503s since they're unrelated, there should simply be another PR that'll introduce coroutines to the versions file

Reply:
Got it - I reverted that change